recipes/ipydatawidgets/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ipydatawidgets" %}
+{% set version = "3.1.0" %}
+{% set sha256 = "973370568799469ed2a6c402f93e2fa8a63c74e044ea1959def3261fb6158464" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >=7,<8
+    - numpy
+    - traittypes >= 0.2
```

Comment:
This should not have a space after the `=`s, which is blocking its conversion.

Could you please send a PR to fix this, @vidartf?

Reply:
Submitted as #6031.

Reply:
Would it be reasonable to have the linter/CI look for these kind of issues?

Reply:
Thanks for doing that.

Yeah. Honestly am surprised `conda-build` didn't reject this. Feel free to raise issues to both `conda-smithy` and `conda-build`.